### PR TITLE
provide favicon

### PR DESCRIPTION
**Pull request by PatriceKammognie**

### CODE REVIEW

File: basyx.aasenvironment/Readme.md [hunk 1]
```diff
@@ -55,3 +55,7 @@ If the AAS Environment file (XML, JSON, or AASX) size exceeds the below mentione
 
 ## AAS Environment Features
 * [AAS Environment Authorization](basyx.aasenvironment-feature-authorization)
+
+
+**Note**
+To configure the favicon , add the favicon.ico to ``basyx-java-server-sdk\basyx.common\basyx.http\src\main\resources\static``.
```
Comment (mdanish98) [inline basyx.aasenvironment/Readme.md:61]: Extra space, link

File: basyx.aasenvironment/Readme.md [hunk 2]
```diff
@@ -55,3 +55,7 @@ If the AAS Environment file (XML, JSON, or AASX) size exceeds the below mentione
 
 ## AAS Environment Features
 * [AAS Environment Authorization](basyx.aasenvironment-feature-authorization)
+
+
+**Note**
```
Comment (mdanish98) [inline basyx.aasenvironment/Readme.md:60]: This should be a title "Configure Favicon"

**Review by mdanish98 — CHANGES_REQUESTED** [review]

File: basyx.aasenvironment/Readme.md (hunk 1 above)
Comment (PatriceKammognie) [reply basyx.aasenvironment/Readme.md]: done

File: basyx.aasenvironment/Readme.md (hunk 2 above)
Comment (PatriceKammognie) [reply basyx.aasenvironment/Readme.md]: done

**Review by mdanish98 — APPROVED** [review]
Approved.

**Pull request merged**